packages/knowledge/src/base.ts

```diff
@@ -588,16 +588,19 @@ export const base: ChartKnowledgeBase = {
 
   treemap: {
     id: 'treemap',
-    name: '',
+    name: 'Treemap',
     alias: [],
-    family: [],
-    def: '',
-    purpose: [],
-    coord: [],
-    category: [],
-    shape: [],
-    dataPres: [],
-    channel: [],
+    family: ['TreeGraph'],
+    def: 'A visual representation of a data tree with nodes. Each node is displayed as a rectangle, sized and colored according to values that you assign',
+    purpose: ['Composition', 'Comparison'],
```

Comment:
```suggestion
    purpose: ['Composition', 'Comparison', 'Hierarchy'],
```

Maybe add 'Hierarchy' as a kind of purposes.